apps/dashboard/src/constants/contracts.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Putting this here solely for the purpose of getting rid of sdk v4
+ */
+export type ContractType =
```

Comment:
ultimately we need to change the usage of this so we can remove it - for now this is fine

Reply:
Yep agreed. I'm aware that ideally we should just remove this. My goal for this one was simply remove as many "import" from /sdk as possible

Reply:
good goal to have overall!